src/Microsoft.AspNet.Cors/CorsOptions.cs

```diff
@@ -0,0 +1,50 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Threading.Tasks;
+using Microsoft.AspNet.Cors.Core;
+
+namespace Microsoft.AspNet.Cors
+{
+    /// <summary>
+    /// Configuration options for <see cref="CorsMiddleware"/>.
+    /// </summary>
+    public class CorsOptions
+    {
+        /// <summary>
+        /// The CORS policy to apply.
+        /// </summary>
+        public ICorsPolicyProvider PolicyProvider { get; set; }
+
+        /// <summary>
+        /// The CORS engine.
+        /// </summary>
+        public ICorsEngine CorsEngine { get; set; }
+
+        /// <summary>
+        /// A policy that allows all headers, all methods, any origin and supports credentials
+        /// </summary>
+        public static CorsOptions AllowAll
```

Comment:
@troydai @blowdart thanks for jumping in :)

Sure. I can remove that. Katana had this option - so I did a similar thing.